Mathlib/Combinatorics/SimpleGraph/Maps.lean

```diff
@@ -266,18 +266,27 @@ def mapDart (d : G.Dart) : G'.Dart :=
 theorem mapDart_apply (d : G.Dart) : f.mapDart d = ⟨d.1.map f f, f.map_adj d.2⟩ :=
   rfl
 
+/-- The graph homomorphism from a smaller graph to a bigger one. -/
+def ofLE (h : G₁ ≤ G₂) : G₁ →g G₂ := ⟨id, @h⟩
```

Comment:
simps?

Reply:
It doesn't generate the lemma I want. See #19860 for an ambitious way to fix `simps` to do the correct thing, but I think the way it is currently designed it won't be easy to change.